.golangci.yml

```diff
@@ -10,15 +10,22 @@ linters:
     - performance
     - sql
     - module
-    - unused
+    - format
+    - import
   enable:
     - predeclared
   disable:
     - maligned # deprecated by fieldalignment
     - scopelint # deprecated by exportloopref
+    - gci # too strict
+    - gofumpt # too strict
+    - contextcheck # too restrictive
 linters-settings:
   govet:
     check-shadowing: true
+  gosec:
+    excludes:
+      - G112 # slowloris attack - TODO (easyfix)
```

Comment:
This is easy fix, will do in a different PR. Just two occurrences but new setting is needed.